core/trino-spi/src/main/java/io/trino/spi/Page.java

```diff
@@ -88,7 +88,7 @@ public long getSizeInBytes()
         if (sizeInBytes < 0) {
             sizeInBytes = 0;
             for (Block block : blocks) {
-                sizeInBytes += block.getLoadedBlock().getSizeInBytes();
+                sizeInBytes += block.getSizeInBytes();
```

Comment:
should we add comment that `Page#getSizeInBytes` returns stale cached value if a lazy block is loaded afterwards? Are there any implications of such staleness in current code?